### ISSUE TITLE
Not able to change the slider values with the keyboard while using NVDA

### ISSUE THREAD

**Issue by VivianSi233** [microsoft/fluentui#22013]
### Environment Information

- **Browser and OS versions**: Edge 99.0.1150.30 Win10

### Describe the issue:

1. Go to https://df.compliance.microsoft.com/insiderriskmgmt/globalsettings?viewid=policies&flight=EnableIRMHorizontalSettings,EnableBetaControls&tid=035c17e3-089f-4f72-9367-043623de09d6
2. Enter the above mentioned URL in Edge dev browser.
3. Insider Risk Management-Settings page will open.
4. Now , Navigate to the 'Policy Timeframe' tab and activate it.
5. Policy Timeframe pane will open.
6. Turn on NVDA and activate it browse mode by pressing 'Insert+Spacebar'.
7. Now, Navigate to the slider which is present on the Activation window.
8. Try to change the slider value in browse mode.

 
-->

#### Actual behavior:

1. In browse mode navigation user unable to change the value of slider by using 'Up/down/Left/Right' keys.
2. There is no information provided for the screen reader users to how to navigate over the sliders.

![image](https://user-images.githubusercontent.com/89379019/157363270-cc95a08d-afe4-4aa3-935f-828c91a82e8f.png)

#### Expected behavior:

1. User should be able to trigger and change the slider values in browse mode navigation.
2. There must be some information should be conveyed by the screen reader which refer how to navigate over the slide and how to change the slider values or any shortcut key information for changing the slider value.

### Documentation describing expected behavior
MAS Reference Link:
[MAS4.2.9-Executions of Available actions](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b71AB7A75-43C8-48FC-AEF7-003D86A03A32%7d&file=MAS%204.2.9%20%E2%80%93%20Execution%20of%20Available%20Actions.docx&wdLOR=c508DAA4C-801C-4A02-AD66-FA4B2A2B1B7E&action=default&mobileredirect=true&cid=1b30bf1c-7ad8-41b3-8c31-34897f02b2f1)

**Comment by smhigley** [comment]
It's generally necessary for screen readers to be in forms mode to modify the value of certain form controls. In this case, it's required because when NVDA is in browse mode, arrow key events are not sent through to the DOM, so there's no way for the page script to handle them and update the slider value.

This is also true of e.g. typing in a text input or interacting with a native `<input type="range">`; you wouldn't be able to change the value in browse mode.
